src/serialbox/core/frontend/stella/SerializationException.h

```diff
@@ -15,11 +15,18 @@
 #ifndef SERIALBOX_CORE_FRONTEND_STELLA_SERIALIZATIONEXCEPTION_H
 #define SERIALBOX_CORE_FRONTEND_STELLA_SERIALIZATIONEXCEPTION_H
 
-#include "serialbox/core/Exception.h"
 #include "serialbox/core/frontend/stella/ForwardDecl.h"
 #include <exception>
 #include <string>
 
+#ifndef SERIALBOX_NOEXCEPT

```

Comment:
Why can't we just include `core/Exception.h`?

Reply:
Because `core/Exception.h` contains C++11 code. This file is #included by STELLA, which is compiled in C++03 mode, so it has to be fully C++03.

Reply:
That would have been my guess. But it means we should remove the check from `core/Exception.h` as it is useless there.

Reply:
Which check?

Reply:
The one which you copied here.

Reply:
Maybe we can remove `SERIALBOX_NOEXCEPT` at all. We can just use `noexcept` all over the place. We need C++11 anyway, and all places where `SERIALBOX_NOEXCEPT` already depend on a C++11 compiler, since they are including a C++11-only header...

Should I go and substitute `SERIALBOX_NOEXCEPT` with `noexcept` everywhere?

Reply:
And just use `throw()` in the stella part of Serialbox...

Reply:
I agree